app/components/navigation_header/index.tsx

```diff
@@ -62,12 +58,17 @@ const NavigationHeader = ({
     const insets = useSafeAreaInsets();
     const styles = getStyleSheet(theme);
 
-    const {largeHeight, defaultHeight} = useHeaderHeight(isLargeTitle, Boolean(subtitle), hasSearch);
+    const {largeHeight, defaultHeight} = useHeaderHeight();
     const containerHeight = useAnimatedStyle(() => {
-        const normal = defaultHeight + insets.top;
-        const calculated = -(insets.top + (scrollValue?.value || 0));
-        return {height: Math.max((normal + calculated), normal)};
-    }, []);
+        const minHeight = defaultHeight + insets.top;
+        const value = -(scrollValue?.value || 0);
+        const height = ((isLargeTitle ? largeHeight : defaultHeight)) + value + insets.top;
+        return {
+            height: Math.max(height, minHeight),
+            minHeight,
+            maxHeight: largeHeight + insets.top + 80,
```

Comment:
Can we name this 80?

Reply:
You can call it Valentina if you want 😎

Sure no problem, I'll assign it to a constant

Reply:
If Valentina is the name that better define its function, yes, let's call it Valentina :laughing: